test/acceptance/testing.go

```diff
@@ -203,6 +210,19 @@ func (c *AccTestCase) terraformApply() error {
 }
 
 func (c *AccTestCase) terraformDestroy() error {
+	if c.RetryDestroy.NumberOfTry <= 0 {
+		return c.doDestroy()
+	}
+
+	r := retrier.New(retrier.ConstantBackoff(c.RetryDestroy.NumberOfTry, c.RetryDestroy.RetryDelay), nil)
```

Comment:
Maybe we can update our existing `acceptance.RetryFor` to be able to use `RetryConfig` instead of adding a new dependency just for this ? I think it is quite overkill to add a dependency for a such simple code.

Reply:
it is not a new dependency we already use retrier in readresource, if it where new it would only be used when compiled the test anyway.
I would have prefere retryfor to use the dependency we already have I think. Also the api is quite good and I don't really see the use to implement something new just for our tests ?

Reply:
Ok, I have seen update on `go.sum` file and in my mind it was a new dependency